waterlog/src/Stylesheets/_sensor.scss

```diff
@@ -0,0 +1,43 @@
+#sensor-diagram {
```

Comment:
inconsistent when compared to `waterlog/src/App.scss`

Reply:
Also add empty line EOF

Reply:
Fixed

Reply:
tabs are the same in both the files, so i only managed to sort out the first one